common/views/components/PrototypeSearchForm/PrototypeSearchForm.tsx

```diff
@@ -86,15 +86,15 @@ const PrototypeSearchForm: FunctionComponent<Props> = ({
   const [, setSearchParamsState] = useSavedSearchState(routeProps);
   const { query } = routeProps;
   const { isEnhanced } = useContext(AppContext);
-
+  const Router = useRouter();
   const searchForm = useRef<HTMLFormElement>(null);
   // This is the query used by the input, that is then eventually passed to the
   // Router
   const [inputQuery, setInputQuery] = useState(query);
   const searchInput = useRef<HTMLInputElement>(null);
   const [portalSortOrder, setPortalSortOrder] = useState(routeProps.sortOrder);
   const [isInitialLoad, setIsInitialLoad] = useState(true);
-
+  const isWorksDetailPage = /works\/[a-zA-Z0-9]+/g.test(Router.asPath);
```

Comment:
I was initially a bit uncomfortable about using the Router for this, because it doesn't feel very scalable. But this component isn't going to be used in many (if any) other places. At least not any time soon. So perhaps this is the most reasonable solution for now. The alternative involves a lot of prop drilling from the page that the component is rendered on, and in retrospect that feels unnecessarily complex for pretty much zero gain.

Reply:
This seems strange to me, why not just remove the component from [here](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/catalogue/webapp/components/Work/Work.tsx#L156-L170)?

Reply:
The issue is that we still want the `SearchTabs` component, but not the filters within.

Reply:
My bad - sorry for misreading that.

Could we have a [`showFilters` prop](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/views/components/SearchForm/SearchForm.js#L29) that we set to false like we do on the [current form]()https://github.com/wellcomecollection/wellcomecollection.org/blob/31b47efd0d99561d85e0c5538dfe18dbf08ea18b/catalogue/webapp/components/Work/Work.tsx#L173?

Reply:
We _could_ (that was my initial thought too – see rambling comment above). The question is, 'is it worth it'?

Reply:
Happy to change this, that SearchTabs.tsx checks the route of the page and sets the filters down to prototypeSearchComponent.

Reply:
Isn't the change adding a prop and passing it through from the two different pages? It feels way more understandable and gets rid of this regex, I'd say it's simpler?

Reply:
> provide some form of util to check what page your on based on route and then you can reference that

You know what page you're on from which page you implement the component?

Reply:
not sure if i can get this where i am from , as the work.tsx check the conditions of the response.    

workResponse.type === 'Work' ||
    workResponse.type === 'Collection' ||
    workResponse.type === 'Section' ||
    workResponse.type === 'Series'

which is controlled by  app.js

 route('/works/:workId/items', '/item', router, app);